frontend/src/components/review/AddReview.vue

```diff
@@ -112,6 +112,23 @@ export default {
                         img: item
                     });
                 });
+
+                let newReview = {};
```

Comment:
This is not ok )
let newReview = {};
newReview.created_at = res.data.created_at;
newReview.description = res.data.description;
newReview.dislikes = 0;
...
This should be inside commit('ADD_REVIEW', review) inside this.addReview.then(...) action. 

Really, vladimir.kudinov is nice man, but we cannot include him everywhere
```
avatar_url: "https://instagram.fhrk1-1.fna.fbcdn.net/vp/f9472104c184408f7e566fce7942d9b3/5BFFDA1C/t51.2885-19/s150x150/18644930_364022684000162_4610752073493905408_a.jpg",
email: "vladimir.kudinov@gmail.com",
```
Remove this and use real data